modules/ob-creating-a-buildah-build.adoc

```diff
@@ -0,0 +1,138 @@
+* You have installed the {builds-operator} on the {ocp-product-title} cluster.
+* You have installed the `buildah` strategy. You can use the following command to verify if the `buildah` strategy is installed:
++ 
+[source,terminal]
+----
+$ oc get cbs | grep buildah
+----
++
+[NOTE]
+====
+If the `buildah` strategy is not installed, see _Installing buildah strategy_ in the _Additional resources_ section.
+====
+
+.Procedure
+
+. Create a `Build` resource and apply it to the {ocp-product-title} cluster:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: shipwright.io/v1beta1
+kind: Build
+metadata:
+  name: buildah-golang-build
+spec:
+  source: <1>
+    git: 
+      url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  strategy: <2>
+    name: buildah
+    kind: ClusterBuildStrategy
+  paramValues: <3>
+  - name: dockerfile
+    value: Dockerfile
+  output: <4>
+    image: image-registry.openshift-image-registry.svc:5000/buildah-example/sample-go-app
+EOF
+----
+<1> The location where the source code is placed.
+<2> The build strategy that you use to build the container.
+<3> A name-value list to specify values for parameters defined in the build strategy. To set the value of the `dockerfile` strategy parameter, specify the Dockerfile location required to build the output image.
+<4> The location where the built image is pushed. By default, the built image is pushed to the {ocp-product-title} cluster internal registry. `buildah-example` is the name of the current project. Ensure that the specified project exists to allow the image push.
+
+. Create a `BuildRun` resource and and apply it to the {ocp-product-title} cluster:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: shipwright.io/v1beta1
+kind: BuildRun
+metadata:
+  name: buildah-golang-buildrun
+spec:
+  build:
+    name: buildah-golang-build <1>
+EOF
+----
+<1> The `spec.build.name` field denotes the respective build to run, which is expected to be available in the same namespace.
++
+The `BuildRun` resource creates a `TaskRun` resource, which then creates the pods to execute build strategy steps, as shown in the following example:
++
+[source,terminal]
+----
+$ oc get br
+NAME                      SUCCEEDED   REASON    STARTTIME   COMPLETIONTIME
+buildah-golang-buildrun   Unknown     Pending   1s
+
+$ oc get tr
+NAME                      	    SUCCEEDED   REASON    STARTTIME   COMPLETIONTIME
+buildah-golang-buildrun-dtrg2     Unknown     Pending     3s
+
+$ oc get tr buildah-golang-buildrun-dtrg2 -ojson | jq .metadata.ownerReferences
+[
+  {
+    "apiVersion": "shipwright.io/v1alpha1",
+    "blockOwnerDeletion": true,
+    "controller": true,
+    "kind": "BuildRun",
+    "name": "buildah-golang-buildrun",
+    "uid": "676dd1da-6ec2-4c04-86b2-cfe5be265260"
+  }
+]
+
+$ oc get pods
+NAME                                READY     STATUS       RESTARTS    AGE
+buildah-golang-buildrun-dtrg2-pod   0/2       Init:0/2     0           9s
+----
```

Comment:
```suggestion
```
We can remove this complete block. This kind of shows up v1alpha1 API, so let's avoid that.. Also, this one's not really mandatory & was just for information.